packages/functional-tests/lib/query-params.ts

```diff
@@ -4,8 +4,15 @@
 
 // This file contains query params that don't reflect states that can be reached from 123done.
 
-export const syncMobileOAuthQueryParams = new URLSearchParams({
+// Minimum needed for OAuth Sync integration
+export const syncMobileBasicQueryParams = new URLSearchParams({
```

Comment:
@tarikeshaq The minimum required param to hit the OAuth integration is a `client_id`. The `context` here tells us it's Sync. When users hit Settings through iOS they'll also give us this client_id, right? Asking since your original test only had context.

Also, this is going to affect desktop oauth Sync as well. However, I feel like that's fine, or at least for right now - when users go through the desktop sync flow, we don't take them to Settings. If we want some parameter differentiator for desktop sync oauth, we could account for it that way, or we could check the user agent on our side (since iPad uses a desktop UA though, it might be tricky to depend on as we've discussed before). Should we account for wanting to distinguish the two before this context is rolled out on the desktop side?

Reply:
Hmm that's a great callout, [mobile will **not** append the client_id when accessing the settings, only the context, (and email and uid if there's a user signed in)](https://github.com/mozilla/application-services/blob/3c43e3fb4a0940d3ee98fe26bf96a5825c160ee6/components/fxa-client/src/internal/mod.rs#L162-L170)!

re: desktop oauth that sounds OK, I can't think of a reason why we'd want to special case desktop oauth from mobile oauth 🤔

Reply:
> Hmm that's a great callout, [mobile will not append the client_id when accessing the settings, only the context, (and email and uid if there's a user signed in)](https://github.com/mozilla/application-services/blob/3c43e3fb4a0940d3ee98fe26bf96a5825c160ee6/components/fxa-client/src/internal/mod.rs#L162-L170)!

Good to know! I'll need to tweak this a little then, hmm. Right now, just supplying that `context` param gives us the 'Web' (default) integration.

> re: desktop oauth that sounds OK, I can't think of a reason why we'd want to special case desktop oauth from mobile oauth 🤔

Cool, that sounds preferable to me anyway. 😁

Reply:
Cross-linking: https://github.com/mozilla/fxa/pull/16244#discussion_r1453681042